libs/sdk/assembly/crypto.ts

```diff
@@ -35,7 +35,7 @@ declare function sign_raw(key_name: ArrayBuffer, clear_text: ArrayBuffer, clear_
 declare function verify_raw(key_name: ArrayBuffer, cipher_text: ArrayBuffer, cipher_text_size: i32, clear_text: ArrayBuffer, clear_text_size: i32): i32;
 // @ts-ignore: decorator
 @external("env", "digest")
-declare function digest_raw(text: ArrayBuffer, text_size: i32, digest: ArrayBuffer, digest_size: i32): i32;
+declare function digest_raw(algorithm: i32, text: ArrayBuffer, text_size: i32, digest: ArrayBuffer, digest_size: i32): i32;
```

Comment:
This is a breaking change for ledger replay. @Gosu14 can you let us know what's the emerging strategy for breaking changes ?

Reply:
Indeed, we will update this one to keep the retro compatibility with the old method signature.